README.md

```diff
@@ -31,7 +31,7 @@ The following RDF triples will be imported once the service starts:
 - [EU political leader name](https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?uri=http://publications.europa.eu/resource/dataset/political-leader&version=20240111-0)
```

Comment:
Not up to date: https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?uri=http://publications.europa.eu/resource/dataset/political-leader&version=20240612-0